zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt

```diff
@@ -65,14 +68,16 @@ class BesluitenApiPlugin(
     fun linkDocumentToBesluit(
         @PluginActionProperty documentUrl: String,
         @PluginActionProperty besluitUrl: String
-    ) {
-        linkDocumentToBesluit(URI(documentUrl), URI(besluitUrl))
-    }
+    ) = linkDocumentToBesluit(URI(documentUrl), URI(besluitUrl))
 
     fun linkDocumentToBesluit(
         documentUrl: URI,
         besluitUrl: URI
+    ) = withLoggingContext(
+        DOCUMENTEN_API.ENKELVOUDIG_INFORMATIE_OBJECT to documentUrl.toString(),
+        BESLUITEN_API.BESLUIT to besluitUrl.toString()
     ) {
+        logger.info { "Linking document $documentUrl to besluit $besluitUrl" }
```

Comment:
I think we should think of a word for this to make it distinct from the valtimo document. maybe something like `ZGW document`?